docs/admins/howto/new-hub.rst

```diff
@@ -25,139 +25,196 @@ The major reasons for making a new hub are:
 
 If your reason is something else, it probably needs some justification :)
 
+Prereqs
+=======
+Working installs of the following utilities:
+  - sops
+  - hubploy
+  - gcloud/kubectl
+  - `cookiecutter <https://github.com/audreyr/cookiecutter>`_
```

Comment:
Is it worthwhile to add info about authorization, i.e. what roles the admin user needs to deploy using the above commands.

Reply:
see https://github.com/berkeley-dsep-infra/datahub/pull/4094/commits/1d76d09f1852cd718bd84c9fdc557f8cc45c5277

Reply:
lemme know if i've missed anything...